### PR TITLE
Fixes Anti-adblock and placeholders.

### PR DESCRIPTION
**Pull request by wout-codes**
Links: 
1. `https://www.prepostseo.com/article-rewriter`
2. `https://smallseotools.com/paraphrasing-tool/`
3. `https://www.check-plagiarism.com/paraphrasing-tool?type=beta`
4. `https://seomagnifier.com/online-paraphrasing-tool`

### CODE REVIEW

File: EnglishFilter/sections/content_blocker.txt [hunk 1]
```diff
@@ -1314,6 +1314,10 @@ topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru
 freemc.host#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72650#issuecomment-771199740
 @@||googletagmanager.com/gtag/js$domain=exe.app
+! anti-adb check-plagiarism.com
+@@||check-plagiarism.com^$generichide
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=check-plagiarism.com
+@@||pagead2.googlesyndication.com/pagead/js/r*/r*/show_ads_impl.js$domain=check-plagiarism.com
```
Comment (Alex-302) [inline EnglishFilter/sections/content_blocker.txt:1320]: better like in all other rules:
`@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$***`
or
`@@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl.js$***`

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1320]: Sure.

Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1320]: The internet is full of anti-adblocks now. Even normal browsing has anti-adblocks like these sites. I went to all these sites to paraphrase, suddenly anti-adblocks came.

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/content_blocker.txt [hunk 2]
```diff
@@ -1317,7 +1317,7 @@ freemc.host#@#.adsbygoogle
 ! anti-adb check-plagiarism.com
 @@||check-plagiarism.com^$generichide
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=check-plagiarism.com
-@@||pagead2.googlesyndication.com/pagead/js/r*/r*/show_ads_impl.js$domain=check-plagiarism.com
+@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$***$domain=check-plagiarism.com
```
Comment (Alex-302) [inline EnglishFilter/sections/content_blocker.txt:1323]: damn

Comment (Alex-302) [reply EnglishFilter/sections/content_blocker.txt:1323]: I mean `***` - is thу second part of the rule)

Comment (wout-codes) [reply EnglishFilter/sections/content_blocker.txt:1323]: Lol. I didn't notice. I blindly copied and pasted.

**Pull request merged**